index.md

```diff
@@ -199,7 +204,7 @@ The Grid instance in our app can be [sorted]({% slug sorting_grid %}) and [paged
 
```

Comment:
_2. Import the necessary types and the data service in `src/app/app.component.ts`:_

how do I know what types do I need to add? Is all of this a "common knowledge"? What about the types that we import from @progress, shouldn't we mention how to identify the required types?

Reply:
Well, I can't explain this well enough, because the types are simply registered in the other Grid documentation as well :)